### PR TITLE
Fix reddit updates

### PR DESCRIPTION
**Pull request by TimJentzsch**
- Add and implement comparable interface for sorting and limiting arrays
- Fix reddit updates not being sent

### CODE REVIEW

File: src/comparable.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+export default interface Comparable<T extends Comparable<T>> {
```
Comment (zachkont) [inline src/comparable.ts]: comparable type extends comparable? What does this do?

Comment (TimJentzsch) [reply src/comparable.ts]: The item you compare to also implements the Comparable interface.

Comment (zachkont) [reply src/comparable.ts]: Hmm this seems weird and maybe too complex? Not sure if there is a better way though. Anyway

Comment (TimJentzsch) [reply src/comparable.ts]: We basically need to do it that way to properly implement the sorting methods. When you want to sort a `Comparable<T>[]` array, you will need to provide a callback function that takes two `Comparable<T>`s as argument. By doing it this way, we can just call the `compareTo` function. 

Maybe there is a cleaner way to accomplish this, but it was the first one I could think of.

**Pull request merged**